### PR TITLE
all: repository got moved to a new github org

### PR DESCRIPTION
**Pull request by mvdan**
Make the appropriate changes.

### CODE REVIEW

**Comment by mvdan** [comment]
I can't get travis.org to recognise the new organisation and repository, which is annoying. Until then, I think we're stuck with no CI builds.

I've done all I can possibly think of and find, so I'll wait until tomorrow and try again. Maybe Travis or GitHub just need a few hours to work their magic.

File: hint.cabal [hunk 1]
```diff
@@ -15,7 +15,7 @@ license:      BSD3
 license-file: LICENSE
 author:       The Hint Authors
 maintainer:   mvdan@mvdan.cc
```
Comment (mvdan) [inline hint.cabal]: If anyone has any ideas on what to do with this, please speak up. I don't think my email should be here - at least not on its own.

Is it possible to say "a changing group of people", or to not give an email address? Without going as far as saying that noone is maintaining the package.

**Comment by psibi** [comment]
Travis seems to be running fine, so probably this can be merge ? Any idea about 8.6 support ? Only the tests seems to need some fix for it to work properly.

**Comment by mvdan** [comment]
I ended up fixing this by having the project on travis-ci.com as opposed to travis-ci.org.

It can indeed be merged, but I was hoping for a review. Not a big deal though, as this change is pretty small.

**Pull request merged**